### PR TITLE
Creation of html and css files

### PR DESCRIPTION
**Pull request by yegonkimutai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# Microverse-start-project
\ No newline at end of file
+# Microverse-start-project
```
Comment (ha-manel) [inline README.md:1]: - [ ] Kindly modify your README file, please use this [template](https://github.com/microverseinc/readme-template) to copy all the sections and modify the placeholder text with information about your project for a professional documentation 👍

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (ha-manel) [inline .gitignore]: - [ ] Kindly add `test.md` to `.gitignore` as mentioned in the project requirements so it doesn't get pushed to the remote repos with the other changes 👍

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @yegonkimutai👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* HTML and CSS files are added successfully 💯 

## Required Changes:

- [ ] Please consider using best practices when you write commit messages, they should be capitalized sentences in the imperative form, example: "Add linters" instead of "added linters" or "linters are added" ... etc 👍 
- [ ] From now on, kindly use better branches names such as: "setup-project", "file-a" doesn't really tell us anything about what's being done in the branch 👍 
- [ ] Kindly update your pull request's title and description to make them descriptive of the changes that have been done in the branch 👍 

_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @joshualyngdoh, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [ ] Kindly check this main changes the previous reviewer requested for and try to make changes on the all.

Highlight 👇 

com

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**